default/scripting/buildings/CULTURE_ARCHIVES.focs.txt

```diff
@@ -17,8 +17,11 @@ BuildingType
             ]
             priority = [[VERY_LATE_PRIORITY]]
             effects = [
-                SetTargetResearch value = Value + (NamedReal name = "BLD_CULTURE_ARCHIVES_TARGET_RESEARCH_FLAT" value = 5)
-                SetTargetIndustry value = Value + (NamedReal name = "BLD_CULTURE_ARCHIVES_TARGET_INDUSTRY_PERPOP" value = (Target.Population / 2))
+                SetTargetResearch value = Value
+                    + (NamedReal name = "BLD_CULTURE_ARCHIVES_TARGET_RESEARCH_FLAT" value = 5)
+                SetTargetIndustry value = Value
+                    + (NamedReal name = "BLD_CULTURE_ARCHIVES_TARGET_INDUSTRY_PERPOP"
+                                 value = (Target.Population / 2))
```

Comment:
I think you can't use a `Target` reference in a named valueref like this.

Reply:
Solved (here in several other places).

Reply:
if i remember rigt i think it might work, but is not officially supported ;)
that is why there are the runtime error messages

of course the pedia should give some error about this, because source/target does not exist.

Reply:
also this breaks the current convention - PERPOP is a value you have to multiply with population and not a value which was already multiplied by population so BLD_CULTURE_ARCHIVES_TARGET_INDUSTRY_PERPOP should probably be 0.5